Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'sprockets', '~> 3.0'
 
 # Application
-gem 'rails', '~> 4.2.2'
+gem 'rails', '~> 4.2'
```

Comment:
Hmm.. Is this what you want? I personally would have not changed this line, so maybe just double check this is correct.